core/api/serializers.py

```
@@ -229,6 +222,21 @@ def update(self, instance, validated_data):
                     **note_data
                 )
 
+    def update(self, instance, validated_data):
```

Ruff (0.8.2)

Missing return type annotation for public function `update`

(ANN201)